apps/twitch-bot/internal/command/helpers/helpers_test.go

```diff
@@ -6,6 +7,87 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestParseMessage(t *testing.T) {
+	type TestCase struct {
+		description    string
+		input          string
+		expectedCmd    string
+		expectedParams []string
+	}
+
+	testCase := []TestCase{
+		{
+			description:    "with just message",
+			input:          "abc",
+			expectedCmd:    "",
+			expectedParams: nil,
+		},
+		{
+			description:    "with a command",
+			input:          "!kampus",
+			expectedCmd:    "kampus",
+			expectedParams: []string{},
+		},
+		{
+			description:    "with wrong mention",
+			input:          "@s",
+			expectedCmd:    "",
+			expectedParams: nil,
+		},
+		{
+			description:    "with just mention",
+			input:          "@senchabot",
```

Comment:
Is this different from the "with wrong mention" test?

Is the length of the mention one of the parameters for our feature?

If with regards to ParseParams "@abcdabcd" and "@a" (or even also a "@") are identical, we don't need to have seperate tests for them